client/installation-helper.cc

```diff
@@ -195,6 +195,39 @@ step4()
     cout << "done" << endl;
 }
 
+void
+step5(const string& description)
+{
+    // fate #317973
+    // step runs in chroot
+
+    // preconditions (maybe incomplete):
+    // snapper rpms installed in chroot
+
+    unsigned int num;
+
+    SCD scd;
+    scd.read_only = false;
+    scd.description = description;
+
+    Snapper snapper("root", "/");
+
+    try
+    {
+        Snapshots::iterator snapshot = snapper.createSingleSnapshot(scd);
```

Comment:
"single" snapshot is OK for installation but for update you will need a pre and post snapshot.